README.Rmd

```diff
@@ -148,187 +71,136 @@ brew install openjdk
-- `get_cell_area(p)` is the area of each planning, in number of cells
-  from the original raster. Note that this area is not necessarily the same
-  for all planning units, as if there are NA cells in the input raster some
-  planning units can partially cover NA and non-NA cells.
+## Usage
 
-We can plot the aggregated data:
+Here we will provide a short tutorial on using the _restoptr R_ package to identify priority areas for restoration. If you haven't already, please install the package (see previous section for installation instructions). We will begin by loading the package.
 
-```{r, out.width="100%", fig.width = 10}
-plot(rast(list(p$data$existing_habitat, p$data$restorable_habitat)), nc = 2,  plg=list(x="topright"))
+```{r "load_package"}
+# load package
+library(restoptr)
 ```
 
-Then, we can add constraints to this base problem. For instance, lets add a
-locked-out constraint, to restrict the number of planning units that can be
-selected for restoration. Such a constraint can be used to account for existing
-land-use practices, feasibility of restoration activities, and stakeholder 
-preferences.
+To identify priorities for restoration, we require information on the spatial location of places that do and do not currently contain suitable habitat. We will now import an example dataset to specify such information (imported as the `habitat_data` object). Specifically, this object is a spatial grid (i.e., raster layer). Each grid cell corresponds to a candidate place for restoration (termed planning unit), and their values indicate the absence or presence of habitat within each planning unit (using values of zero and one, respectively).
 
-```{r}
-locked_out_data <- rast(
- system.file("extdata", "locked_out.tif", package = "restoptr")
+```{r "habitat_data", out.width = "100%", fig.width = 10}
+# import data
+habitat_data <- rast(
+  system.file("extdata", "habitat_hi_res.tif", package = "restoptr")
 )
-p <- p %>% add_locked_out_constraint(data = locked_out_data)
-```
 
-We can also add a constraint on the amount of restored area that is allowed by the
-project:
+# preview data
+print(habitat_data)
 
-```{r}
-p <- p %>% add_restorable_constraint(90, 220, unit = "ha")
+# visualize data
+plot(habitat_data, plg = list(x = "topright"))
 ```
 
-And finally a compactness constraint, which limits the spatial extent of the
-selected restoration area:
-
-```{r}
-p <- p %>% add_compactness_constraint(2.4, unit = "km")
-```
+Restoration efforts are often limited in terms of the places where they can be implemented. For instance, restoration efforts may not be feasible in dense cities. To specify which planning units are not feasible for restoration, we will import another example dataset (imported as the `locked_out_data` object). The grid cell values in this raster layer indicate which planning units should be considered available for restoration or not (using values of zero and one, respectively).
```

Comment:
Just like the previous comment, maybe we could consider adding a context sentence:

"In our example dataset, locked out areas correspond to areas that cannot be accessed using the existing track network"

Reply:
Awesome - I'll add that in too.